bot/discordbot.py

```
@@ -156,8 +165,12 @@
         )
     }
     await get_verified_channel.edit(overwrites=overwrites_verified_channel)
-    await get_verified_channel.send("""To gain full access to this Discord server, please enter '/verify' in the chat box to initiate the verification process. Rest assured, we do not retain any of your private information during this process. If you encounter any issue, please contact humanID at discord@human-id.org. Replies to this message do not reach humanID.
-                                """)
+    status_code = await validate_server(guild)
+    if status_code != 400:
+        await get_verified_channel.send(f"""To gain full access to this Discord server, please enter '/verify' in the chat box to initiate the verification process. Rest assured, we do not retain any of your private information during this process. If you encounter any issue, please contact humanID at discord@human-id.org. Replies to this message do not reach humanID.
+                                    """)
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)